### PR TITLE
Add vehicle models

### PR DESCRIPTION
**Pull request by ygnn123**

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: models/intel/vehicle-detection-0203/accuracy-check.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+models:
+  - name: vehicle-detection-0203
+
+    launchers:
+      - framework: dlsdk
```
[the remark this answers is not in the document]
Comment (eaidova) [reply models/intel/vehicle-detection-0203/accuracy-check.yml:5]:
```suggestion
      - framework: dlsdk
        adapter:
               type: class_agnostic_detection
               scale: ???
```

[the remark this answers is not in the document]
Comment (ygnn123) [reply models/intel/vehicle-detection-0203/accuracy-check.yml:5]: scale: ??? What is "???"?

[the remark this answers is not in the document]
Comment (eaidova) [reply models/intel/vehicle-detection-0203/accuracy-check.yml:5]: never mind, your model do not use scale, so this line will be enough:
```suggestion
      - framework: dlsdk
        adapter: class_agnostic_detection
```

File: models/intel/vehicle-detection-0203/accuracy-check.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        adapter:
+           type: class_agnostic_detection
+
+    datasets:
+      - name: crossroad_extra_untagged_vehicle
```
Comment (Anna-Mironova) [inline models/intel/vehicle-detection-0203/accuracy-check.yml:10]: Labels are numbered from zero in this dataset, but in the prediction, labels are equal to one. Please add a new dataset(suggestion below) and use it in these cases.

` 
 - name: crossroad_extra_untagged_vehicle_labels_from_1
    data_source: crossroad_extra_untagged/val_data
    annotation_conversion:
      converter: mscoco_detection
      annotation_file: crossroad_extra_untagged/val_coco_no_bg_vehicle.json
      has_background: True
    annotation: crossroad_extra_untagged_vehicle_labels_from_1.pickle
    dataset_meta: crossroad_extra_untagged_vehicle_labels_from_1.json
`

File: tools/accuracy_checker/dataset_definitions.yml [hunk 3]
```diff
@@ -413,7 +413,16 @@ datasets:
...
     annotation: crossroad_extra_untagged_vehicle.pickle
     dataset_meta: crossroad_extra_untagged_vehicle.json
-
+  
+  - name: crossroad_extra_untagged_vehicle_labels_from_1
+    data_source: crossroad_extra_untagged/val_data
+    annotation_conversion:
+    converter: mscoco_detection
+    annotation_file: crossroad_extra_untagged/val_coco_no_bg_vehicle.json
+    has_background: True
```
Comment (Anna-Mironova) [inline tools/accuracy_checker/dataset_definitions.yml:422]:
```suggestion
      converter: mscoco_detection
      annotation_file: crossroad_extra_untagged/val_coco_no_bg_vehicle.json
      has_background: True
```

File: models/intel/vehicle-detection-0203/description/vehicle-detection-0203.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+Average Precision (AP) is defined as an area under
+the [precision/recall](https://en.wikipedia.org/wiki/Precision_and_recall)
+curve.
+
+## Performance
+
+## Inputs
+
+Name: `input`, shape: [1x3x800x1333] - An input image in the format [BxCxHxW],
```
Comment (Anna-Mironova) [inline models/intel/vehicle-detection-0203/description/vehicle-detection-0203.md:29]:
```suggestion
Name: `input`, shape: [1x3x800x1344] - An input image in the format [BxCxHxW],
```

File: models/intel/vehicle-detection-0204/description/vehicle-detection-0204.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+Average Precision (AP) is defined as an area under
+the [precision/recall](https://en.wikipedia.org/wiki/Precision_and_recall)
+curve.
+
+## Performance
+
+## Inputs
+
+Name: `input`, shape: [1x3x800x1333] - An input image in the format [BxCxHxW],
```
Comment (Anna-Mironova) [inline models/intel/vehicle-detection-0204/description/vehicle-detection-0204.md:29]:
```suggestion
Name: `input`, shape: [1x3x800x1344] - An input image in the format [BxCxHxW],
```

File: models/intel/vehicle-detection-0205/description/vehicle-detection-0205.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+Average Precision (AP) is defined as an area under
+the [precision/recall](https://en.wikipedia.org/wiki/Precision_and_recall)
+curve.
+
+## Performance
+
+## Inputs
+
+Name: `input`, shape: [1x3x800x1333] - An input image in the format [BxCxHxW],
```
Comment (Anna-Mironova) [inline models/intel/vehicle-detection-0205/description/vehicle-detection-0205.md:29]:
```suggestion
Name: `input`, shape: [1x3x800x1344] - An input image in the format [BxCxHxW],
```

**Comment by eaidova** [comment]
Jenkins ok to test

**Comment by ygnn123** [comment]
Jenkins please retry a build

**Comment by eaidova** [comment]
@ygnn123 , here linting issues, please fix:
```
08:52:32  + /home/jenkins/workspace/omz/precommit/accuracy_checker_ubuntu18/venv/bin/yamllint configs/vehicle-detection-0203.yml
08:52:32  configs/vehicle-detection-0203.yml
08:52:32    1:64      error    no new line character at the end of file  (new-line-at-end-of-file)
08:52:32  
08:52:32  [Pipeline] sh
08:52:32  + /home/jenkins/workspace/omz/precommit/accuracy_checker_ubuntu18/venv/bin/yamllint configs/vehicle-detection-0204.yml
08:52:32  configs/vehicle-detection-0204.yml
08:52:32    1:64      error    no new line character at the end of file  (new-line-at-end-of-file)
08:52:32  
08:52:32  [Pipeline] sh
08:52:33  + /home/jenkins/workspace/omz/precommit/accuracy_checker_ubuntu18/venv/bin/yamllint configs/vehicle-detection-0205.yml
08:52:33  configs/vehicle-detection-0205.yml
08:52:33    1:64      error    no new line character at the end of file  (new-line-at-end-of-file)
```
regarding docs CI failure, please ignore it, it is not connected with your changes

**Review by Anna-Mironova — APPROVED** [review]

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Review by eaidova — APPROVED** [review]

**Comment by IRDonch** [comment]
Jenkins please retry a build

**Pull request merged**